docs/csharp/language-reference/language-specification/toc.md

```diff
@@ -0,0 +1,152 @@
+# [Introduction - * C# 6.0 draft specification* ](index.md)
+# [Lexical structure](../../../../_csharplang/spec/lexical-structure.md)
```

Comment:
I usually prefer when you go to a sub-toc to have a parent node and others as H2 headers or lower.